botorch/utils/sampling.py

```diff
@@ -878,17 +907,26 @@ def get_polytope_samples(
     if equality_constraints:
         dense_equality_constraints = sparse_to_dense_constraints(
             d=bounds.shape[-1],
-            constraints=equality_constraints,
+            constraints=normalize_linear_constraints(bounds, equality_constraints)
+            if normalize_bounds
+            else equality_constraints,
         )
     else:
         dense_equality_constraints = None
     polytope_sampler = HitAndRunPolytopeSampler(
         inequality_constraints=dense_inequality_constraints,
-        bounds=bounds,
+        bounds=torch.tensor(
+            [list(torch.zeros(bounds.shape[-1])), list(torch.ones(bounds.shape[-1]))]
+        ).to(bounds)
+        if normalize_bounds
+        else bounds,
```

Comment:
would do sth like this for better readability:
```
normalized_bounds  = torch.zeros_like(bounds)
normalized_bounds[1, :] = 1.0
polytope_sampler = HitAndRunPolytopeSampler(...
```

Reply:
Good idea.